src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Task {
```

Comment:
>  Write descriptive header comments for all public classes/methods.

Missing javadoc for `Task` and methods, but I believe you will address this while doing the `A-JavaDoc` level